addons/cilium/enable

```diff
@@ -96,20 +96,33 @@ else
   echo "Waiting for cilium. This may take several minutes."
   "$SNAP/microk8s-kubectl.wrapper" -n $NAMESPACE rollout status ds/cilium
 
-  # Fetch the Cilium CLI binary and install
-  ARCH=$(arch)
-  SCRIPT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
-  mkdir -p "${SCRIPT_DIR}/cli/"
-  fetch_as "https://github.com/cilium/cilium-cli/releases/download/v0.15.2/cilium-linux-${ARCH}.tar.gz" "${SCRIPT_DIR}/cli/cilium.tar.gz"
-  tar -zxf "${SCRIPT_DIR}/cli/cilium.tar.gz" -C "${SCRIPT_DIR}/cli"
-  chmod +x "${SCRIPT_DIR}/cli/cilium"
-  cp "${SCRIPT_DIR}/cilium" "${SNAP_COMMON}/plugins/cilium"
-  chmod +x "${SNAP_COMMON}/plugins/cilium"
+  # Skip installing the Cilium CLI binary if the cluster is in offline mode
+  if [[ "$*" != *"--offline"* ]]; then
+    # Fetch the Cilium CLI binary and install
+    ARCH=$(arch)
+    SCRIPT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+    mkdir -p "${SCRIPT_DIR}/cli/"
+    fetch_as "https://github.com/cilium/cilium-cli/releases/download/v0.15.2/cilium-linux-${ARCH}.tar.gz" "${SCRIPT_DIR}/cli/cilium.tar.gz"
+    tar -zxf "${SCRIPT_DIR}/cli/cilium.tar.gz" -C "${SCRIPT_DIR}/cli"
+    chmod +x "${SCRIPT_DIR}/cli/cilium"
+    cp "${SCRIPT_DIR}/cilium" "${SNAP_COMMON}/plugins/cilium"
+    chmod +x "${SNAP_COMMON}/plugins/cilium"
+  else
+    echo "Cluster is in offline mode. Skipping installing Cilium binary."
+  fi
 
   echo "Cilium is enabled"
   echo
-  echo "You can now enable hubble with:"
-  echo "  sudo microk8s cilium hubble enable"
-
+  echo "You can now enable hubble relay and UI with:"
+  echo "  sudo microk8s helm3 upgrade cilium cilium/cilium \\"
+  echo "    --version ${CILIUM_VERSION} \\"
+  echo "    --namespace ${NAMESPACE} \\"
+  echo "    --reuse-values \\"
+  echo "    --set hubble.relay.enabled=true \\"
+  echo "    --set hubble.ui.enabled=true"
+  echo
+  echo
+  echo "In offline mode, the Cilium CLI binary will not be installed."
+  echo "You can enable offline mode with:"
+  echo "  sudo microk8s enable cilium --offline"
```

Comment:
I'm not sure if we should give this message after enable succeeds, seems like a help message would be more appropriate. For now it isn't too important I'd say.